src/components/DashStat.container.js

```diff
@@ -21,7 +22,7 @@ function DashStatTotalCaptures(props) {
   const [total, setTotal] = useState(null);
 
   const getTotal = async () => {
-    console.log('-- dash getTotal');
+    log.debug('-- dash getTotal');
     const { count } = await api.getCaptureCount(activeFilter);
```

Comment:
The standard dashboard cards no longer work.
They will soon be replaced by the new Org cards, but we'll need to keep them live for the moment.